### ISSUE TITLE
Math plugin won't work after introducing #document property to DomConverter (and other classes)

### ISSUE THREAD

**Issue by pomek** [ckeditor/ckeditor5#6362]
In this issue – https://github.com/ckeditor/ckeditor5/issues/6091 – we change our API. All breaking changes are described in the PR – https://github.com/ckeditor/ckeditor5-engine/pull/1826#issue-377772902.

When I was working on the all-features sample, I wanted to add Math plugin to the editor. My all branches were checked out on changes related to #6091 and unfortunately, Math plugin crashed.

It threw an error:
![image](https://user-images.githubusercontent.com/2270764/75659805-de0e7680-5c6a-11ea-8add-81141aeba7ce.png)

I went through their sources and what I can notice:

* `@wiris/ckeditor5-math/src/conversion/mathmldataprocessor.js`, it extends our `XmlDataProcessor` which changed its API. The following fix should help:

    ```js
	export default class MathmlDataProcessor extends XmlDataProcessor {
	
		/**
		 * Creates a new instance of the MathML data processor class.
		 *
		 * @param {module:engine/view/document~Document} document
		 * @param {Object} options Configuration options.
		 * @param {Array<String>} [options.namespaces=[]] A list of namespaces allowed to use in the XML input.
		 */
		constructor( document, options = {} ) {
	
			// Call XmlDataProcessor's constructor
			super( document, options );
	
			// Use BR_FILLER instead of NSBP_FILLER to identify <br/> and remove them when retrieving data
			this._domConverter = new DomConverter( document, { blockFiller: BR_FILLER } );
		}
    ```

* `@wiris/mathtype-ckeditor5/src/conversion/downcast.js` - L31 creates an instance of `HtmlDataProcessor`. This class requires an instance of `view.Document` now. It could be taken from `conversionApi`: 

    ```js
    const htmlDataProcessor = new HtmlDataProcessor( conversionApi.writer.document );
    ```

* In the same file as above, L32 (`CustomMathmlDataProcessor`) – the same fix.
* `@wiris/mathtype-ckeditor5/src/plugin.js` - L176 - missing the view document:

    ```js
    editor.data.processor = new CustomMathmlDataProcessor( editor.editing.view.document );
    ```

* `@wiris/mathtype-ckeditor5/src/integration.js` – L144, as above:

    ```js
    const mathmlDP = new CustomMathmlDataProcessor( writer.document );
    ```

I am not sure I found everything thought.

**Comment by mlewand** [comment]
Also @pomek listed changed in MathType's `ckeditor5-demo` {which is a branch for the upcoming rewrite) https://github.com/wiris/html-integrations/compare/ckeditor5-demo...pomek:ckeditor5-demo

**Comment by mlewand** [comment]
MathType team has recently released a beta `7.19.0-beta.0` release and it's working fine :+1
